types/aws-lambda/index.d.ts

```diff
@@ -75,8 +78,10 @@ export interface CustomAuthorizerEvent {
     methodArn: string;
     authorizationToken?: string;
     headers?: { [name: string]: string };
+    multiValueHeaders?: { [name: string]: string[] };
     pathParameters?: { [name: string]: string } | null;
     queryStringParameters?: { [name: string]: string } | null;
+    multiValueQueryStringParameters?: { [name: string]: string[] } | null;
```

Comment:
Just noticed these inconsistent `| null` here. Gross, as all these properties are all certainly going to be one of not present or `null` if they even are optional.
That said, not your problem!